controls/treeview/control-template.md

```diff
@@ -0,0 +1,96 @@
+---
+title: Control Template
+page_title: .NET MAUI TreeView Documentation - Control Template
+description: Review what is the default control template of the Telerik .NET MAUI TreeView control and all elements we have in the template. 
+position: 7
+slug: treeview-control-template
+---
+
+# .NET MAUI TreeView Control Template
+
+The Telerik UI for .NET MAUI TreeView control has a set of elements inside the `ControlTemplate`. These elements are `CheckBox`, `Image`, `Expand/Collapse` indicator and the `Label`. 
```

Comment:
The introductory sentence must provide the following answers, preferably in this order:
1. What is a `ControlTemplate`?
2. What do I use it for?
3. When do I use it?
4. Why don't I use something else?